docs/apm/api.asciidoc

```diff
@@ -708,3 +709,87 @@ curl -X DELETE "http://localhost:5601/api/apm/sourcemaps/apm:foo-1.0.0-644fd5a9"
 --------------------------------------------------
 {}
 --------------------------------------------------
+
+////
+*******************************************************
+*******************************************************
+////
+
+[role="xpack"]
+[[agent-key-api]]
+=== Agent Key API
+
+The Agent Key API allows you to configure agent keys to authorize requests from agents to the APM Server.
```

Comment:
I think we should make it clear we are talking about APM agents as opposed to the Elastic agent

```suggestion
The Agent Key API allows you to configure agent keys to authorize requests from APM agents to the APM Server.
```